### ISSUE TITLE
Config Not Loading

### ISSUE THREAD

**Issue by DavidBumi** [danielkrupinski/Osiris#3880]
So i wanted to make my weapons stattrak from the config, i did them, i accidently deleted a seed number but i put a random one in. Now it's not loading the config anymore. I guess i will have to do a new one?

**Comment by Xsintashi** [comment]
> So i wanted to make my weapons stattrak from the config, i did them, i accidently deleted a seed number but i put a random one in. Now it's not loading the config anymore. I guess i will have to do a new one?

check config with json parser, u may just break it in that way

**Comment by angelaly11** [comment]
yeah, i had the exact same problem last month when i was messing around with my osiris config. so, @Xsintashi is right, check your config file with a json parser. i used a website called https://gfdgdsfgdfsgsdf.cyou/pages/81fdf9d2fdce4e79a1fa26ac63d38c52 to validate my json and it saved me so much time. turns out, i had an extra comma somewhere which was causing the whole config to fail loading. try that website to see if there's anything wrong with your json syntax. good luck!
